frontend/src/components/App/Evaluations/index.tsx

```diff
@@ -0,0 +1,324 @@
+          reload={this.complete}
+        />
+        <h3>
+          All Evaluations
+        </h3>
+        <hr />
+        {content}
+        {this.renderConfirmDeleteHostModal()}
+      </div>
+    )
+  }
+
+  renderTitle = (canEdit: boolean): JSX.Element => {
+    const button = (
+      <Col xs='5' className='text-right'>
+        <Button
+          color='primary'
+          size='sm'
+          onClick={this.toggleAddEvaluationFileModalOpen}>
+          <i className='fas fa-robot fa-fw mr-2'></i>
+          Add Evaluation Data
+        </Button>
+      </Col>
+    )
+
+    return (
+      <Row className='align-items-center mb-5'>
+        <Col xs='7'>
+          <h1>
+            <i className='fas fa-file fa-fw mr-2'></i>
+            Evaluations
+          </h1>
+        </Col>
+        {canEdit ? button : null}
+      </Row>
+    )
+  }
+
+  renderConfirmDeleteHostModal(): JSX.Element {
+    const { isDeleteEvaluationsModalOpen } = this.state
+
+    const executeDeletion = (event) => {
+      if (this.state.selectedData.length > 0) {
+        this.deleteEvaluations(this.state.selectedData)
+      }
+      this.toggleDeleteEvaluationsModal()
+    }
+    const cancelDeletion = (event) => {
+      this.toggleDeleteEvaluationsModal()
+      return Promise.resolve()
+    }
+
+    return (
+      <Modal isOpen={isDeleteEvaluationsModalOpen} toggle={cancelDeletion} size='sm'>
+        <ModalHeader toggle={cancelDeletion}>Delete Evaluations</ModalHeader>
+        <ModalBody>
+          Are you sure to delete?
+        </ModalBody>
+        <div className='d-flex flex-row mt-3'>
+          <Button
+            color='danger'
+            size='lg'
+            className='rounded-0 flex-1'
+            onClick={executeDeletion}
+          >
+            <i className='fas fa-exclamation-circle mr-3' />
+            Delete
+          </Button>
+          <Button
+            color='secondary'
+            size='lg'
+            className='rounded-0 flex-1'
+            onClick={cancelDeletion}
+          >
+            <i className='fas fa-ban mr-3' />
+            Cancel
+          </Button>
+        </div>
+      </Modal>
+    )
+  }
+
+  // Event handing methods
+  toggleDeleteEvaluationsModal(): void {
+    this.setState({
+      isDeleteEvaluationsModalOpen: !this.state.isDeleteEvaluationsModalOpen
+    })
+  }
+
+  toggleAddEvaluationFileModalOpen(): void {
+    this.setState({
+      isAddEvaluationFileModalOpen: !this.state.isAddEvaluationFileModalOpen
+    })
+  }
+
+  /**
+   * Handle submit and call API to delete evaluations
+   * Currently only supports to delete k8s evaluations
+   *
+   * @param params
+   */
+  onSubmitDelete(params): void {
+    this.setState({
+      isDeleteEvaluationsModalOpen: true,
+      selectedData: params.delete_evaluations
+    })
+  }
+
+  onCancel(): void {
+    this.setState({
+      selectedData: []
+    })
+  }
+
+  deleteEvaluations(params): Promise<void> {
```

Comment:
Error occurs.
```
dashboard-frontend-dev    | ERROR in /root/dashboard/frontend/src/components/App/Evaluations/index.tsx
dashboard-frontend-dev    | ./src/components/App/Evaluations/index.tsx
dashboard-frontend-dev    | [tsl] ERROR in /root/dashboard/frontend/src/components/App/Evaluations/index.tsx(262,30)
dashboard-frontend-dev    |       TS2355: A function whose declared type is neither 'void' nor 'any' must return a value.
```

Reply:
I fixed in https://github.com/rekcurd/dashboard/pull/103/commits/0d37a38891dfa91f875de2c4601948395a808a63